crates/bevy_gizmos/src/lib.rs

```diff
@@ -90,6 +90,10 @@ pub struct GizmoPlugin;
 
 impl Plugin for GizmoPlugin {
     fn build(&self, app: &mut bevy_app::App) {
+        // Gizmos cannot work without either a 3D or 2D renderer.
+        #[cfg(all(not(feature = "bevy_pbr"), not(feature = "bevy_sprite")))]
+        bevy_utils::tracing::error!("bevy_gizmos requires either bevy_pbr or bevy_sprite. Please enable one as an optional dependency.");
```

Comment:
Bevy doesn't usually use `error!` directly, it usually imports it.

Reply:
I forgot that `bevy_log` existed and `bevy_utils::tracing` was the first thing that popped up in autocomplete. This will require me adding `bevy_log` as a dependency in `Cargo.toml`.

Reply:
Fixed in fbb6cefe18690f26b0fb398508f636fb26409519. It feels weird adding a new dependency just for one (rare) case, but `tracing` is being pulled in already by `bevy_utils`.